engine/src/test/java/org/camunda/bpm/engine/test/api/identity/IdentityServiceTest.java

```diff
@@ -1066,6 +1067,44 @@ public void shouldCreateUserWithEmptyUserId() {
     assertThat(user).isNotNull();
   }
 
+  @Test
+  public void shouldIncludePlaintextPasswordInUserToString() {
```

Comment:
Just wondering, should this be named `shouldNot...`? 🤔 
```suggestion
  public void shouldNotIncludePlaintextPasswordInUserToString() {
```